packages/staking/src/features/overview/Overview.tsx

```diff
@@ -35,7 +35,7 @@ export const Overview = () => {
 
   if (
     !totalCoinBalance ||
-    !protocolParameters?.stakeKeyDeposit ||
+    !protocolParameters?.hasOwnProperty('stakeKeyDeposit') ||
     !balancesBalance?.available?.coinBalance ||
     !rewardAccounts
   ) {
```

Comment:
I was concerned about why this works but `coinBalance` is a `string` not `number`.

Can we make this snippet a standard for now? Much more readable imo.

```suggestion
    if (
      [
        totalCoinBalance,
        protocolParameters?.stakeKeyDeposit,
        balancesBalance?.available?.coinBalance,
        rewardAccounts
      ].some(isNil)
    ) {
```

Reply:
Let's keep the scope only to the original issue with protocol parameters please.